app/views/layouts/application.html.erb

```diff
@@ -67,6 +63,12 @@
         <%= link_to 'Cart', carts_path, class: "btn" %>
         <span class="badge badge-secondary"><%= @cart.total_count %></span>
       </li>
+      <% if current_user %>
```

Comment:
@mimilettd I think that this is an appropriate use of conditionals in the view! Rock on with your socks on!!!!